client/components/NavigationDrawer/NavigationDrawerOpener.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC, useState, useCallback } from 'react'
+
+import Crowi from 'client/util/Crowi'
+import Icon from 'components/Common/Icon'
+import NavigationDrawer from 'client/components/NavigationDrawer/NavigationDrawer'
+
+interface Props {
+  crowi: Crowi
+}
+
+const NavigationDrawerContainer: FC<Props> = ({ crowi }) => {
```

Comment:
This component's name is different from filename.
I think it should be same name `NavigationDrawerContainer` or `NavigationDrawerOpener`.

Reply:
Ah, good catch! I renamed it to `NavigationDrawerContainer`.